### ISSUE TITLE
panic: strconv.Atoi: parsing "": invalid syntax

### ISSUE THREAD

**Issue by tomasdeml** [jesseduffield/lazygit#549]
After upgrading to v0.11.2 lazygit started crashing during startup:

```
panic: strconv.Atoi: parsing "": invalid syntax

goroutine 1 [running]:
github.com/jesseduffield/lazygit/pkg/commands.mustConvertToInt(...)
        /go/src/github.com/jesseduffield/lazygit/pkg/commands/patch_modifier.go:151
github.com/jesseduffield/lazygit/pkg/commands.(*GitCommand).GetTags.func1(0x1ae, 0x1c0, 0xc00044ed01)
        /go/src/github.com/jesseduffield/lazygit/pkg/commands/loading_tags.go:62 +0x5c8
sort.medianOfThree_func(0xc00044eff0, 0xc0000cd240, 0x1ae, 0x1c0, 0x1d2)
        /usr/local/go/src/sort/zfuncversion.go:58 +0xb8
sort.doPivot_func(0xc00044eff0, 0xc0000cd240, 0x1ae, 0x1d3, 0xc, 0x1d4)
        /usr/local/go/src/sort/zfuncversion.go:80 +0x8f
sort.quickSort_func(0xc00044eff0, 0xc0000cd240, 0x1ae, 0x1d3, 0xe)
        /usr/local/go/src/sort/zfuncversion.go:143 +0xa1
sort.quickSort_func(0xc00044eff0, 0xc0000cd240, 0x0, 0x1eb, 0x11)
        /usr/local/go/src/sort/zfuncversion.go:148 +0x13a
sort.Slice(0xb04c80, 0xc0000cd220, 0xc00044eff0)
        /usr/local/go/src/sort/slice.go:17 +0xf8
github.com/jesseduffield/lazygit/pkg/commands.(*GitCommand).GetTags(0xc00017dce0, 0x1c80, 0xc000062900, 0x122a160, 0x0, 0xc000097078)
        /go/src/github.com/jesseduffield/lazygit/pkg/commands/loading_tags.go:41 +0x2bd
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).refreshTags(0xc0002e2000, 0x0, 0x0)
        /go/src/github.com/jesseduffield/lazygit/pkg/gui/tags_panel.go:60 +0x3a
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).refreshBranches(0xc0002e2000, 0xc00020cf20, 0x100000004, 0xc0002e2248)
        /go/src/github.com/jesseduffield/lazygit/pkg/gui/branches_panel.go:83 +0x57
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).refreshSidePanels(0xc0002e2000, 0xc00020cf20, 0x0, 0xc0001a2240)
        /go/src/github.com/jesseduffield/lazygit/pkg/gui/view_helpers.go:17 +0x40
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).loadNewRepo(0xc0002e2000, 0xc27157, 0x8)
        /go/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:669 +0xc8
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).onInitialViewsCreation(0xc0002e2000, 0xc00020cf20, 0xc28e39)
        /go/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:659 +0xb9
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).layout(0xc0002e2000, 0xc00020cf20, 0x3a012300356000, 0x2030000)
        /go/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:599 +0x1292
github.com/jesseduffield/gocui.ManagerFunc.Layout(0xc00021cd80, 0xc00020cf20, 0x0, 0x8)
        /go/pkg/mod/github.com/jesseduffield/gocui@v0.3.1-0.20191116013947-b13bda319532/gui.go:388 +0x37
github.com/jesseduffield/gocui.(*Gui).flush(0xc00020cf20, 0xc000097b88, 0x4439c5)
        /go/pkg/mod/github.com/jesseduffield/gocui@v0.3.1-0.20191116013947-b13bda319532/gui.go:505 +0x346
github.com/jesseduffield/gocui.(*Gui).MainLoop(0xc00020cf20, 0xc000000003, 0xc000097ce8)
        /go/pkg/mod/github.com/jesseduffield/gocui@v0.3.1-0.20191116013947-b13bda319532/gui.go:413 +0x5b
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).Run(0xc0002e2000, 0x0, 0x0)
        /go/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:821 +0x4fe
github.com/jesseduffield/lazygit/pkg/gui.(*Gui).RunWithSubprocesses(0xc0002e2000, 0xc00017d960, 0xc00017d8f0)
        /go/src/github.com/jesseduffield/lazygit/pkg/gui/gui.go:830 +0x36
github.com/jesseduffield/lazygit/pkg/app.(*App).Run(0xc00017d7a0, 0xc0000d0700, 0xc00017d7a0)
        /go/src/github.com/jesseduffield/lazygit/pkg/app/app.go:166 +0x72
main.main()
        /go/src/github.com/jesseduffield/lazygit/main.go:73 +0x77f
```

**To Reproduce**
1. Start lazygit
2. See error

**Expected behavior**
No crash.

**Desktop (please complete the following information):**
 - OS: Windows 10 1803
 - Lazygit Version 0.11.2 (v0.11.1 is also affected, v0.10.6 works fine)

**Comment by jesseduffield** [comment]
whoops looks like I was being a little too strict when sorting tags with version numbers. This PR should fix it https://github.com/jesseduffield/lazygit/pull/551

**Comment by jesseduffield** [comment]
should be fixed in v0.11.3, let me know otherwise :)
